frontend/public/components/overview/project-overview.tsx

```diff
@@ -304,6 +311,18 @@ const ProjectOverviewListItem = connect<
             <ControllerLink controller={current} />
           </>
         )}
+        {firingAlerts.length > 0 && (
+          <>
+            &nbsp; &nbsp;
```

Comment:
Can we add spacing here using CSS, so that this extra fragment will not be required.